internal/integration/integration_test.go

```diff
@@ -56,6 +56,49 @@ func TestVariables(t *testing.T) {
 	assert.Zero(t, resp.User)
 }
 
+func TestInterfaceNoFragments(t *testing.T) {
+	_ = `# @genqlient
+	query queryWithInterfaceNoFragments($id: ID!) {
+		being(id: $id) { __typename id name }
+		me { id name }
+	}`
+
+	ctx := context.Background()
+	server := server.RunServer()
+	defer server.Close()
+	client := graphql.NewClient(server.URL, http.DefaultClient)
+
+	resp, err := queryWithInterfaceNoFragments(ctx, client, "1")
+	require.NoError(t, err)
+
+	assert.Equal(t, "1", resp.Me.Id)
+	assert.Equal(t, "Yours Truly", resp.Me.Name)
+
+	user, ok := resp.Being.(*queryWithInterfaceNoFragmentsBeingUser)
```

Comment:
Is there a plan to add helpers so callers don't need to look up the generated type for the type switch? e.g.
```
user, ok := resp.BeingUser()  // or UserBeing()
```

Reply:
Yep!  There's a TODO somewhere, which is now fixed in #57.